### ISSUE TITLE
[REQUEST] add NUMPAD_ENTER for picking

### ISSUE THREAD

**Issue by BaumschulenElrik** [wong2/pick#111]
[REQUEST]

Would you be so kind and enable the the NUMPAD_ENTER key to finish the selection just like ENTER?
Thanks alot! Great little module!

**Comment by ScadeBlock** [comment]
Yeah, I have tried this module before and that's greet! But seems this module isn't often upgrade and missing some features. So I made another module - [BetterPick](https://github.com/ScadeBlock/BetterPick) here. Maybe you can give it a try. Thanks!
